config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  get '/users/', to: 'users#index', as: 'users'
+  get '/users/:id', to: 'users#show', as: 'user'
+
+  get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  get '/users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
```

Comment:
OPTIONAL
- [ ] For route, also you can use resources and only keywords. 

![image](https://user-images.githubusercontent.com/38635158/171268880-6c52e06c-31bd-4e50-b154-c06f170e8435.png)